.ansible-lint

```diff
@@ -44,7 +44,10 @@ use_default_rules: true
 # rulesdir:
 #   - ./rule/directory/
 
-# Ansible-lint completely ignores rules or tags listed below
+# Ansible-lint is able to recognize and load skip rules stored inside
+# `tests/sanity/ignore.txt` files (same as ansible-test). To skip
```

Comment:
Since this is unrelated to the `ansible-test` `sanity` test, I don't think it makes sense to put it in the `tests/sanity/` directory.

For the same reason, naming the file `ignore.txt` (especially in the directory used by `ansible-test`) could be confusing to users, since the name is so close to the one used by `ansible-test`. There's also the potential for further confusion since the file formats are different.

Reply:
The references to `tests/sanity/ignore.txt` and `ansible-test` should be removed.